extensions/javascript/schemas/jsconfig.schema.json

```diff
@@ -9,42 +9,132 @@
 					"type": "boolean"
 				},
-				"locale": {
-					"description": "The locale to use to show error messages, e.g. en-us.",
+				"emitBOM": {
+					"description": "When down-level compiling, emit a UTF-8 Byte Order Mark (BOM) in the beginning of output files.",
+					"type": "boolean"
+				},
+				"inlineSourceMap": {
+					"description": "When down-level compiling, emit a single file with source maps instead of having a separate file.",
+					"type": "boolean"
+				},
+				"inlineSources": {
+					"description": "When down-level compiling, emit the source alongside the sourcemaps within a single file; requires --inlineSourceMap to be set.",
+					"type": "boolean"
+				},
+				"jsx": {
+					"description": "When down-level compiling, specify JSX code generation: 'preserve' or 'react'.",
+					"enum": [
+						"preserve",
+						"react"
+					]
+				},
+				"reactNamespace": {
+					"description": "When down-level compiling, specifies the object invoked for createElement and __spread when targeting 'react' JSX emit.",
 					"type": "string"
 				},
+				"listFiles": {
+					"description": "When down-level compiling, print names of files part of the compilation.",
+					"type": "boolean"
+				},
 				"mapRoot": {
-					"description": "Specifies the location where debugger should locate map files instead of generated locations",
+					"description": "When down-level compiling, specifies the location where debugger should locate map files instead of generated locations",
 					"type": "string",
 					"format": "uri"
 				},
 				"module": {
-					"description": "Specify used module type: 'commonjs', 'system', 'umd', 'es6', or 'es2015'. Note: 'amd' is currently not supported.",
+					"description": "When down-level compiling, specify module code generation: 'commonjs', 'system', 'umd', 'amd', 'es6', or 'es2015'.",
 					"enum": [
 						"commonjs",
 						"umd",
+						"amd",
 						"system",
 						"es6",
 						"es2015"
 					]
 				},
-
+				"newLine": {
+					"description": "When down-level compiling, specifies the end of line sequence to be used when emitting files: 'CRLF' (dos) or 'LF' (unix).",
+					"enum": [
+						"CRLF",
+						"LF"
+					]
+				},
+				"noEmit": {
+					"description": "When down-level compiling, do not emit output.",
+					"type": "boolean"
+				},
+				"noEmitHelpers": {
+					"description": "When down-level compiling, do not generate custom helper functions like __extends in compiled output.",
+					"type": "boolean"
+				},
+				"noEmitOnError": {
+					"description": "When down-level compiling, do not emit outputs if any type checking errors were reported.",
+					"type": "boolean"
+				},
 				"noLib": {
 					"description": "Do not include the default library file (lib.d.ts).",
 					"type": "boolean"
 				},
+				"noResolve": {
+					"description": "When down-level compiling, do not add triple-slash references or module import targets to the input files.",
```

Comment:
I changed it from
Do not add triple-slash references or module import targets to the *list of compiled* files.